lib/rust/ensogl/app/theme/hardcoded/src/lib.rs

```diff
@@ -198,7 +196,7 @@ define_themes! { [light:0, dark:1]
                     section_heading_size = 16.0, 16.0;
                     section_heading_offset = 50.0, 50.0;
                     section_heading_text_offset = 13.0, 13.0;
-                    section_heading_font = "Causten-Semibold", "Causten-Semibold";
+                    section_heading_font = "default", "default";
```

Comment:
Those should be constants + I don't think "default" is proper here. If we have style, we have a concrete font in mind. The default is meant only as a fallback if the intended font does not exist.

Reply:
As agreed via phone, we keep it this way. The reason for it is that we are slowly preparing for exporting themes to `json` files and we want to have a notion of a "default" font there.